src/templates/base.html

```diff
@@ -163,21 +163,19 @@
           const query_type = document.getElementById('query-type');
           const query_input = document.getElementById('query-input');
 
-          //do nothing if search bar is empty
```

Comment:
Assuming the code you have included below is functionally the same as this code, it would be good to retain this comment.